editor/src/messages/tool/common_functionality/shape_editor.rs

```diff
@@ -187,6 +188,11 @@ impl ShapeEditor {
 		self.iter(document).flat_map(|shape| shape.manipulator_groups().iter())
 	}
 
+	// Sets the selected points to all points for the corresponding intersection
+	pub fn select_all_anchors(&self, responses: &mut VecDeque<Message>, itersections: Vec<u64>) {
```

Comment:
Note that anchors are not the same as manipulator points - anchors are just the square widgets whereas manipulator points include both the circles and the squares.

Reply:
Thanks for clarifying. The name uses anchors, however the body of the function calls the operation SelectAllManipulatorPoints. Would this need to be changed to some other operation or function that selects all anchors since the issue's description mentions anchors not manipulator points

Reply:
Just from testing this branch's behavior, it looks like when you use the Path tool to click the filled region of a shape, only the anchors do get selected, not the handles. So from a behavior point of view, does the correct thing, assuming I understand correctly what's being talked about here. If any names of functions or messages don't reflect their functionality, please feel empowered to rename those. (In general, as you run across code, always don't hesitate to improve code quality along the way, as long as it's not massively disruptive amounting to a full refactor itself— a separate branch for really big changes would be preferred, while the same branch is fine for smaller changes.)

Reply:
Ok thanks! I believe it's just a renaming of the operation I mentioned above. I made it at the start of this branch.